src/backend/vulkan/src/device.rs

```diff
@@ -2044,6 +2019,45 @@ impl super::Device {
 
         Ok((swapchain, images))
     }
+
+    pub unsafe fn image_view_from_raw(
+        &self,
+        raw_image: vk::Image,
+        view_type: vk::ImageViewType,
+        format: format::Format,
+        swizzle: format::Swizzle,
+        usage: image::Usage,
+        range: image::SubresourceRange,
+    ) -> Result<n::ImageView, image::ViewCreationError> {
```

Comment:
I would have preferred adding a `Device::image_from_raw()` method, but in my case it is OpenXR that creates the Vulkan images and it hides some details needed to construct gfx images.